OCPBUGS-24557: Explain spec.clusterImageSetNameRef and spec.clusters.clusterImageSetNameRef in SiteConfig CR (#71724)
openshift/openshift-docs: modules/ztp-sno-siteconfig-config-reference.adoc

Adding additional components back into the system might require additional reserved CPU capacity.
====

|`spec.clusters.clusterImageSetNameRef`

> |References the cluster image set that is used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at site level.

Reviewer: ```suggestion
|Specifies the cluster image set used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at the site level.
```
- Same comment as above re `specifies` vs `references`
- Edit for minimalism
